docs/guides/tools/proposer-based-timestamps-runbook.md

```diff
@@ -4,25 +4,27 @@ order: 3
 
 # Proposer-Based Timestamps Runbook
 
-Version v1.0 of CometBFT added new constraints for the timestamps included in
-each block created by CometBFT. The new constraints mean that validators may
-fail to produce valid blocks or may issue `nil` prevotes for proposed blocks
-depending on the configuration of the validator's local clock.
+From version `v1.0`, CometBFT has new constraints for the timestamps included
+in produced blocks.
+
+The new constraints mean that validators may fail to produce valid blocks,
+which causes other validators to issue `nil` prevotes, thus rejecting the
+proposed block, depending on the configuration of the validator's local clock.
 
 ## What is this document for?
 
 This document provides a set of actionable steps for application developers and
 node operators to diagnose and fix issues related to clock synchronization and
-configuration of the [`SynchronyParams`](https://github.com/cometbft/cometbft/blob/main/spec/core/data_structures.md#synchronyparams) consensus parameters.
+configuration of the [`SynchronyParams`](../../explanation/core/proposer-based-timestamps.md#consensus-parameters) consensus parameters.
 
 Use this runbook if you observe that validators are frequently voting `nil` for a block that the rest
 of the network votes for, or if validators are frequently producing block proposals
 that are rejected by the rest of the network.
 
 ## Requirements
 
-To use this runbook, you must be running a node that has the [Prometheus metrics endpoint enabled](../core/metrics.md)
-and the [RPC endpoint](../core/rpc.md) enabled and accessible.
+To use this runbook, you must be running a node that has the [Prometheus metrics endpoint enabled](../../explanation/core/metrics.md)
+and the [RPC endpoint](../../explanation/core/rpc.md) enabled and accessible.
```

Comment:
Please remove the leading "the " in line 41 below

Reply:
Ah, found it.